src/coreclr/vm/method.hpp

```diff
@@ -581,7 +581,7 @@ class MethodDesc
     //
     // RequiresInstMethodDescArg()
     //    The method is itself generic and is shared between generic
-    // instantiations but is not itself generic.  Furthermore
+    // instantiations but is not itself generic. WTF LOL. Furthermore
```

Comment:
Words are hard, and copy/pasting is easy. The "but is not itself generic" is just wrong and should be deleted from this comment.